packages/arb-token-bridge-ui/tests/e2e/cypress.d.ts

```diff
@@ -42,6 +42,9 @@ declare global {
         tokenAddress: string
       }): typeof searchAndSelectToken
       fillCustomDestinationAddress(): typeof fillCustomDestinationAddress
+      typeAmount(
+        amount: string | number
+      ): Cypress.Chainable<JQuery<HTMLElement>>
```

Comment:
I would use `typeAmount: typeof typeAmount` instead, so it's updated as `typeAmount` is changed